### PR TITLE
*: improve more metrics

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
At present, it's hard to know the process status of making replicas for offline or down stores. We might not know why the operator is failed to be created.

### What is changed and how it works?
This PR adds a new metrics named `processStatusCounter` to record the failure reason when creating the operator for the offline or down store. Also, this PR renames some labels to make them more clear.

Pros and cons:
Pros: 
1. We can know why the operator cannot be created and the kind of failure reasons.
2. After #1746, once we encounter "no-store", we can directly find some information from filter metrics.

Cons:
1. It records some duplicated information with `checkerCounter`.

**UPDATE:
According to the comments, reuse `checkerCounter`.**

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: server/checker/metrics.go [hunk 1]
```diff
@@ -23,8 +23,17 @@ var (
 			Name:      "event_count",
 			Help:      "Counter of checker events.",
 		}, []string{"type", "name"})
+
+	processStatusCounter = prometheus.NewGaugeVec(
```
Comment (nolouch) [inline server/checker/metrics.go]: use `checkerCouter`?

Comment (lhy1024) [reply server/checker/metrics.go]: checkerCounter?

File: server/schedulers/random_merge.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ func (s *randomMergeScheduler) Schedule(cluster schedule.Cluster) []*operator.Op
 	stores := cluster.GetStores()
 	store := s.selector.SelectSource(cluster, stores)
 	if store == nil {
-		schedulerCounter.WithLabelValues(s.GetName(), "no-store").Inc()
+		schedulerCounter.WithLabelValues(s.GetName(), "no-source-store").Inc()
```
Comment (lhy1024) [inline server/schedulers/random_merge.go]: is it necessary to change "no-adjacent" to "no-target-store" in line 82?

**Pull request merged**